drivers/sensor/tsl2591/Kconfig

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2023 Kurtis Dinelle
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig TSL2591
+	bool "OSRAM-AMS TSL2591 light sensor"
+	default y
+	depends on DT_HAS_AMS_TSL2591_ENABLED
+	select I2C
+	help
+	  Enable driver for TSL2591 sensor.
+
+if TSL2591
+config TSL2591_FETCH_WAIT
+	bool "Wait for valid sensor reading before fetch"
+	default n
+	help
+	  If set, the driver will automatically wait for the duration of an integration cycle
+	  during a fetch call if necessary.
+
+config TSL2591_WARN_SATURATED
+	bool "Warn if sensor is potentially saturated"
+	default n
```

Comment:
Remove

Reply:
Fixed, and rebased with merge conflict fixed.